phys2bids/viz.py

```diff
@@ -267,3 +267,5 @@ def plot_all(ch_name, timeseries, units, freq, infile, outfile, dpi=SET_DPI, siz
     outfile = os.path.join(outfile, os.path.splitext(os.path.basename(infile))[0] + ".png")
     LGR.info(f"saving channel plot to {outfile}")
     fig.savefig(outfile, dpi=dpi, bbox_inches="tight")
+    fig.clf()
```

Comment:
I'm not sure the clf is necessary if you are closing the figures. But I need to double check myself.
```suggestion
```